test/multiSamlStrategy.js

```diff
@@ -218,12 +214,10 @@ describe('strategy#generateServiceProviderMetadata', function() {
   it('passes options on to saml strategy', function(done) {
     var passportOptions = {
       passReqToCallback: true,
-      authnRequestBinding: 'HTTP-POST',
```

Comment:
I feel like this is a reasonable thing to be able to test for, especially since the default is 'redirect'.

Reply:
Sure, but this is testing against what was a Strategy option, but is now simply a SAML config option.  

I guess I could add it to the "'uses given options to setup internal saml provider" test